aten/src/ATen/native/transformers/hip/flash_attn/flash_api.hip

```diff
@@ -0,0 +1,634 @@
+// !!! This is a file automatically generated by hipify!!!
```

Comment:
If the file is automatically generated by hipify then why are you checking it in?

Reply:
Fixed in 264dd0a337f5fdabf66d45b7817653541f1b85ea